src/commands/go-build.yaml

```
@@ -3,6 +3,11 @@ parameters:
     type: "string"
   os:
     type: "string"
+  path:
+    default: "."
+    type: "string"
+    description: |
+      The path to the Go package to build.
```

What do you think about?

```suggestion
      The path where the Go package to build is located.
```